Sources/LiveKit/core/SignalClient.swift

```diff
@@ -269,21 +269,25 @@ extension SignalClient {
         sendRequest(r)
     }
 
-    func sendUpdateTrackSettings(sid: String, disabled: Bool, videoQuality: Livekit_VideoQuality) {
+    func sendUpdateTrackSettings(sid: String,
+                                 disabled: Bool,
+                                 width: Int = 0,
+                                 height: Int = 0) {
         logger.debug("[SignalClient] Sending update track settings")
 
         let r = Livekit_SignalRequest.with {
             $0.trackSetting = Livekit_UpdateTrackSettings.with {
                 $0.trackSids = [sid]
                 $0.disabled = disabled
-                $0.quality = videoQuality
+                $0.width = 0
```

Comment:
Should these be set to `width` and `height` from the arguments?

Reply:
omg oniiichann yess !!! 😄🙏

Reply:
🙇